### ISSUE TITLE
Clarify distinction between hosted versions of the game

### ISSUE THREAD

**Issue by jeremyckahn** [jeremyckahn/farmhand#214]
Farmhand is hosted from both https://jeremyckahn.github.io/farmhand/ and https://www.farmhand.life/. Because they are hosted on different domains and game save files are based on domain-dependent Indexed DB instances, save files are not shared between the two. This can be worked around by exporting from one to the other.

Task: Add an in-game note about this save file incompatibility between domains and provide instructions for working around it.

**Comment by lstebner** [comment]
still want to do this? maybe we can just write a page on the wiki instead of something in game?

**Comment by jeremyckahn** [comment]
I don't think this is valuable enough to keep the issue open. We can revisit in the future if necessary!
